src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -103,7 +103,7 @@ protected override void CheckBase(BindingDiagnosticBag diagnostics)
             }
 
             // you need to know all bases before you can ask this question... (asking this causes a cycle)
-            if (this.IsGenericType && !baseContainsErrorTypes && this.DeclaringCompilation.IsAttributeType(localBase))
```

Comment:
>AllowGenericAttributes [](http://example.com/codeflow?start=59&length=22)

Can you check this method was removed if no longer used?

Reply:
It looks like it was removed here, it just didn't show in CodeFlow for some reason: https://github.com/dotnet/roslyn/pull/26337/commits/b54d0f82ba63109b8d6086d003d84612fcee98ee#diff-eeb711a0e114d9141bc8954eb3835d844496716984be379289c71bee68f571e8